pandas/core/groupby/groupby.py

```diff
@@ -180,6 +180,16 @@ class providing the base-class of operations.
         A callable that takes a {input} as its first argument, and
         returns a dataframe, a series or a scalar. In addition the
         callable may take positional and keyword arguments.
+    include_groups : bool, default True
+        When True, will attempt to apply ``func`` to the groupings in
+        the case that they are columns of the DataFrame. If this raises a
+        TypeError, the result will be computed with the groupings excluded.
```

Comment:
Can you describe also what `include_groups=False` does.

Reply:
Done